typify-impl/src/lib.rs

```diff
@@ -269,6 +276,7 @@ impl TypeSpaceSettings {
 
     /// Replace a referenced type with a named type. This causes the referenced
     /// type *not* to be generated.
+    /// Overwrites prior replacements with the same type name.
```

Comment:
```suggestion
    /// If the same `type_name` is specified multiple times, the last one is honored.
```

Wrap with previous line.